src/main/java/org/apache/jackrabbit/filevault/maven/packaging/FormatDocviewXmlMojo.java

```diff
@@ -109,7 +111,14 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             getLog().info("Skip executing mojo 'format-xml' for incremental builds as parameter 'enableForIncrementalBuilds' is set to 'false'");
         } else {
             File jcrSourceDirectory = AbstractSourceAndMetadataPackageMojo.getFirstExistingDirectory(jcrRootSourceDirectory);
-            executeInternal(jcrSourceDirectory);
+            if (jcrSourceDirectory != null) {
+                executeInternal(jcrSourceDirectory);
+            } else {
+                if (getLog().isDebugEnabled()) {
+                    getLog().debug("None of the configured jcrRootSourceDirectory directories exists, skipping: " +
```

Comment:
I would issue a WARN because that may also hint at a misconfiguration. Also please use SLF4J placeholders.

Reply:
Maybe one could check the current packaging to silently ignore this for packaging != content-package

Reply:
`org.apache.maven.plugin.logging.Log` does not support placeholder replacement, thats why the check if `isDebugEnabled()`

Other Mojos may do the configuration validation. I don't see this being necessary in the bounded context for "format xml". So said the debug log level is correct from my perspective as "if the directory does not exist, do nothing". 

This change addressees the execution of the `format-xml` goal from command line and even if unlikely it may not run in a maven project at all. So I would rather not depend on the packaging here.

Reply:
Makes sense, agreed.